shared_model/backend/protobuf/transaction_responses/proto_tx_response.hpp

```diff
@@ -85,6 +108,23 @@ namespace shared_model {
       // stub hash
       const Lazy<crypto::Hash> hash_{
           [this] { return crypto::Hash(this->proto_->tx_hash()); }};
+
+      /**
+       * @return priority this transaction response
+       */
+      int priority() const noexcept {
+        return iroha::visit_in_place(
```

Comment:
`return variant_->which()` ?

Reply:
No, this won't work, because then we should place classes in variant according to their priorities, and this is not as obvious as this `visit_in_place` function IMHO